### ISSUE TITLE
Add support for importing Google Authenticator accounts

### ISSUE THREAD

**Issue by b2loo** [X1nto/Mauth#71]
I'm trying to import from a QR code generated in the Google Authenticator, and neither choosing from an image or scanning with my camera will work.

- Choosing from image brings me back to the empty list
- Scanning from camera does not do anything, no matter how clear the screen I'm scanning from is in the camera viewport.

I love the design of this app and was really looking forward to switching, but this is a hard block for me to be able to start using it.

**Comment by X1nto** [comment]
Are you trying to import the big QR code that's generated when exporting all accounts?

**Comment by b2loo** [comment]
> Are you trying to import the big QR code that's generated when exporting all accounts?

nope, just one at a time. I've had issues with exporting multiple at once before so I haven't tried it.

**Comment by X1nto** [comment]
One at a time should work. Is it broken for all QR codes or just some of them?

**Comment by b2loo** [comment]
> One at a time should work. Is it broken for all QR codes or just some of them?

I tried doing both this morning. It won't recognize anything at all.

**Comment by X1nto** [comment]
I'll take a look

**Comment by b2loo** [comment]
> I'll take a look

thank you very much!
For additional information;

I'm running LineageOS 21-20241103-NIGHTLY-shiba
(Android 14 AP2A.240905.003)
on the Google Pixel 8

I'm not rooted, or have any firmware tweaks or Xposed Modules of any kind that might interfere

if you need any more Information please let me know

**Comment by X1nto** [comment]
Found the cause. Google Authenticator uses a different way to export the accounts. I'll add support for it

**Comment by b2loo** [comment]
> Found the cause. Google Authenticator uses a different way to export the accounts. I'll add support for it

Awesome, thank you very much
I eagerly await the update

**Comment by nxs-neo** [comment]
Please add import support. I really love the ui of your app but the only thing stopping me is not being able to import. I've been following this project for a long time and this is the only feature I really want but keeps getting delayed.

**Comment by X1nto** [comment]
Google Auth Import will be supported in the next major version. I won't release 0.10.0 without it.

**Comment by X1nto** [comment]
Added in v0.10.0. Reopen if you detect any issues.

**Comment by ThunderArtist** [comment]
I just tried to import my Google Authenticator QR code and it didn't work. I've tried both image select (with the code screenshotted) and camera scan modes.

Version 0.10.0 from F-Droid (on Android 13).
Let me know if you need any other info.
